app/src/question/styles.js

```diff
@@ -25,6 +25,26 @@ module.exports = {
     color: color.green
   },
 
+  '.vote-tally-bar': {
+    height:'9px',
```

Comment:
copy and pasting is bad, bc now if you want to change the height you have to do it 4 times.